odoo_self_order_details/static/src/components/product_details_pop_up/product_patch.js

```diff
@@ -0,0 +1,42 @@
+import { patch } from "@web/core/utils/patch";
+import { ProductCard } from "@pos_self_order/app/components/product_card/product_card";
+import { ProductDetailsPopup } from "./product_details_popup";
+
+patch(ProductCard.prototype, {
+    async selectProduct(qty  = 1){
```

Comment:
```suggestion
    async selectProduct(qty){
```

Make sure `super.selectProduct(qty)` is called

Reply:
done!